docs/developers/plugins.md

```diff
@@ -52,7 +52,7 @@ Chart.plugins.register({
 
 ### Plugin ID
 
-Plugins must define a unique id in order to be configurable.
+Plugins must define an unique id in order to be configurable.
```

Comment:
This was actually correct before. https://www.quora.com/Which-is-the-correct-grammar-usage-a-unique-or-an-unique